scripts/aap-automation/install-aap.sh

```
@@ -0,0 +1,375 @@
+    log_error "Failed to retrieve admin password"
+    exit 1
+fi
+
+# Get route URL
+log_info "Retrieving AAP Platform route..."
+ROUTE_URL=$(oc get route ${PLATFORM_NAME} -n $AAP_NAMESPACE -o jsonpath='{.spec.host}' 2>/dev/null || echo "")
+
+if [ -z "$ROUTE_URL" ]; then
+    log_warn "Route not found yet. Waiting for route to be created..."
+    sleep 30
+    ROUTE_URL=$(oc get route ${PLATFORM_NAME} -n $AAP_NAMESPACE -o jsonpath='{.spec.host}' 2>/dev/null || echo "")
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)